Tools/BasicStation-Certificates-Generation/helper-functions

```diff
@@ -50,42 +50,42 @@ Arguments:
 }
 
 ecdsaKey() {
-    openssl ecparam -name prime256v1 -genkey | openssl ec -out $1 > /dev/null 2>&1
+    openssl ecparam -name prime256v1 -genkey | openssl ec -outform $2 -out $1 > /dev/null 2>&1
 }
 
 rootCA() {
     mkdir -p $2
-    ecdsaKey $2/"${1,,}"-ca.key
+    ecdsaKey $2/"${1,,}"-ca.key DER
     # When using git bash, any argument starting with a forward slash is interpreted as a path.
     # Prefixing with "//" escapes that, but openssl will then skip the first segment interpreted as '/OU'
     # That's why first segment is '/SKIP=X'
-    openssl req -new -key $2/"${1,,}"-ca.key -out $2/"${1,,}"-ca.csr -subj "//SKIP=X/OU=StarterKit/O=StarterKit/C=CH/CN=$1 Root CA" > /dev/null 2>&1
-    openssl x509 -req -set_serial 1 -days 365 -in $2/"${1,,}"-ca.csr -signkey $2/"${1,,}"-ca.key -out $2/"${1,,}"-ca.crt > /dev/null 2>&1
+    openssl req -new -key $2/"${1,,}"-ca.key -keyform DER -out $2/"${1,,}"-ca.csr -subj "//SKIP=X/OU=StarterKit/O=StarterKit/C=CH/CN=$1 Root CA" > /dev/null 2>&1
+    openssl x509 -req -set_serial 1 -days 365 -in $2/"${1,,}"-ca.csr -signkey $2/"${1,,}"-ca.key -keyform DER -out $2/"${1,,}"-ca.crt -outform DER
     rm $2/"${1,,}"-ca.csr
 }
 
 servercert() {
     mkdir -p $3
-    ecdsaKey $3/$1.key
+    ecdsaKey $3/$1.key PEM
```

Comment:
nit shouldn't we switch everythin to der?

Reply:
unfortunately, creating a DER cert is not allowing us the possibility to directly convert from DER to PFX. We would anyways require a DER to PEM to PFX. That's why I kept it like this